test/core/models/ocelot/LabelSpec.scala

```diff
@@ -455,4 +468,64 @@ class LabelSpec extends BaseSpec with ProcessJson {
       labels3.value("Location") shouldBe Some("Here")
     }
   }
+
+  "Labels Flow stack" must {
+    "Allow adding a Flow to top of stack" in {
+      val labels = LabelCache().pushFlows(Seq("1","2"), "3", Some("loop"), Seq("One", "Two", "Three"), Nil)
```

Comment:
Should we have three values for "labelValues" here? I would have thought the number of label values should match the number of next values minus the value used for continue.

Reply:
There is always an extra next element, this is where to go when all the flows have completed and each flow corresponds with a labelValue, in the case of a Sequence its the continuation next